OmsAgent/omsagent.py

```diff
@@ -150,7 +150,7 @@ def verifyShellBundleSigningAndChecksum():
         raise Exception("Unable to find the dscgpgkey.asc file at " + dscGPGKeyFilePath)
 
     importGPGKeyCommand = "sh ImportGPGkey.sh " + dscGPGKeyFilePath
-    exit_code, output = run_command_with_retries_output(importGPGKeyCommand, retries = 0, retry_check = retry_skip)
+    exit_code, output = run_command_with_retries_output(importGPGKeyCommand, retries = 0, retry_check = retry_skip, check_error = False)
```

Comment:
should check `exit_code` here?

Reply:
We don't care about the exit_code  at that place because I do more check down below.